mezzanine/utils/urls.py

```diff
@@ -133,5 +133,13 @@ def path_to_slug(path):
     for prefix in (lang_code, settings.SITE_PREFIX, PAGES_SLUG):
         if prefix:
             path = path.replace(prefix, "", 1)
-    path = path.strip("/") if settings.APPEND_SLASH else path.lstrip("/")
+    path = clean_slashes(path)
     return path or "/"
```

Comment:
`return path or "/"` here should move to `clean_slashes` - that will allow for a homepage slug (eg "/") to be set from admin